lib/completion.dart

```diff
@@ -81,9 +81,8 @@ class DartCompleter extends CodeCompleter {
         if (delta > 0 && delta <= text.length) {
           text = text.substring(delta);
         }
-
         // TODO: Use classes to decorate the completion UI ('cm-builtin').
-        return new Completion(text, displayString: displayString);
+        return new Completion(text, displayString: displayString, type: "type-"+completion.type.toLowerCase());
```

Comment:
We should only pass in the `type` named param if the `completion.type` field is not not null (not all completions will have the `element` info).